src/style/variables.less

```diff
@@ -1,4 +1,5 @@
 @header-height: 3.5rem; // 56px
+@app-main-height: @header-height + 2.5rem; // 96px for banner
```

Comment:
nit: I don't think this needs to be extracted to a variable, as it's only used in 1 place.

I do like having the comment there for explaining the extra `2.5rem` though, good idea.

Reply:
makes sense, I had pushed the change prior to seeing your comment, I'll use `calc` directly within `index.less`

Reply:
Sounds good, thanks!